numaprom/tools.py

```diff
@@ -131,14 +118,14 @@ def get_data(file_path: str) -> Any:
     return data
 
 
-def decode_msg(msg: bytes) -> Dict[str, Any]:
+def decode_msg(msg: bytes) -> Optional[Dict[str, Any]]:
     msg = msg.decode("utf-8")
 
     try:
         data = json.loads(msg)
     except Exception as ex:
         LOGGER.exception("Error in Json serialization: %r", ex)
-        return ""
+        return None
```

Comment:
Should we rethrow the json load failure and let the caller determine the decoding failure strategy?